functions/index.js

```diff
@@ -938,29 +941,71 @@ exports.checkUserStatus = functions.runWith(runWithObj).pubsub.schedule("* * * *
   // OFFLINE, HBBack new => IDLE
   const queryRefForOffline = usersColRef.where("status", "==", "OFFLINE").where("heartbeatBackground", ">=", T);
   queryRefForOffline.get().then((querySnapshot) => {
-    console.log('checkUserStatus, queryRefForOffline', querySnapshot.size);
+    console.log("checkUserStatus, queryRefForOffline", querySnapshot.size);
     querySnapshot.forEach(async (queryDocSnapshotUser) => {
-      console.log('checkUserStatus, queryRefForOffline, queryDocSnapshotUser.id', queryDocSnapshotUser.id);
+      console.log("checkUserStatus, queryRefForOffline, queryDocSnapshotUser.id", queryDocSnapshotUser.id);
       const p = queryDocSnapshotUser.ref.update({status: "IDLE"});
       ps.push(p);
     });
   });
 
-  console.log('checkUserStatus, ps', ps.length);
+  console.log("checkUserStatus, ps", ps.length);
 
   return Promise.all(ps);
 });
+// commented un used code due to lint errors
+// const sendALGO = async (algodclient, fromAccountAddr, toAccountAddr, signAccount, amount, wait = false) => {
+//   // txn
+//   const suggestedParams = await algodclient.getTransactionParams().do();
+//   const transactionOptions = {
+//     from: fromAccountAddr,
+//     to: toAccountAddr,
+//     amount,
+//     suggestedParams,
+//   };
+//   const txn = algosdk.makePaymentTxnWithSuggestedParamsFromObject(
+//       transactionOptions,
+//   );
+//
+//   // sign
+//   const signedTxn = txn.signTxn(signAccount.sk);
+//
+//   // send raw
+//   try {
+//     const {txId} = await algodclient.sendRawTransaction(signedTxn).do();
+//     console.log("txId", txId);
+//
+//     // confirm
+//     if (wait) {
+//       const timeout = 5;
+//       await waitForConfirmation(algodclient, txId, timeout);
+//       console.log("sendALGO, confirmed");
+//     }
+//
+//     return {
+//       txId,
+//       error: null,
+//     };
+//   } catch (e) {
+//     console.log("error", e);
+//     return {
+//       txId: null,
+//       error: e,
+//     };
+//   }
+// };
 
-const sendALGO = async (algodclient, fromAccountAddr, toAccountAddr, signAccount, amount, wait = false) => {
+const sendASA = async (algodclient, fromAccountAddr, toAccountAddr, signAccount, amount, assetIndex, wait = false) => {
```

Comment:
why was this renamed?

Reply:
I didn't change it. It was from below revision c84521a555c0013a5c985ab7749884a3a9c9ace0

in this commit due to link spaces and unused code was giving error that why some unwanted changes have been done.